src/blocks/pyllis-wb-2/scripts.js

```diff
@@ -0,0 +1,157 @@
+const trackListing = document.getElementsByClassName('track-link');
+
+const nowPlaying = document.querySelector('.now-playing');
+const trackArt = document.querySelector('.track-art');
+const trackName = document.querySelector('.track-name');
+const trackArtist = document.querySelector('.track-artist');
+
+const playPauseBtn = document.querySelector('.play-pause-track');
+const nextBtn = document.querySelector('.next-track');
+const prevBtn = document.querySelector('.prev-track');
+
+const seekSlider = document.querySelector('.seek-slider');
+const currTime = document.querySelector('.current-time');
+const totalDuration = document.querySelector('.total-duration');
+const volumeSlider = document.querySelector('.volume-slider');
+
+let trackIndex = 0;
+let isPlaying = false;
+let updateTimer;
+
+let currTrack = document.createElement('audio');
+
+const trackList = [
+  {
+    id: 1,
+    name: 'Scorpio Harvest',
+    artist: 'DJ Chai',
+    image: 'https://audio-app-pw.s3.amazonaws.com/scorpioharvest.jpg',
+    path: 'https://audio-app-pw.s3.amazonaws.com/Chai_ScorpioHarvest2015.mp3'
+  },
+  {
+    name: 'FnF Fall Classic',
+    artist: 'DJ Chai',
+    image: 'https://audio-app-pw.s3.amazonaws.com/flower.jpg',
+    path: 'https://audio-app-pw.s3.amazonaws.com/FnF_Fall_Classic.mp3'
```

Comment:
Please provide the licenses to the music & images you use. Are these under public domain?

Reply:
All images are under public domain, and music is owned by myself, also under public domain.